src/service-override/environment.ts

```diff
@@ -20,8 +20,7 @@ function getServiceOverride (): IEditorOverrideServices
 
 function getServiceOverride (options?: IWorkbenchConstructionOptions): IEditorOverrideServices {
   return {
-    [IEnvironmentService.toString()]: new SyncDescriptor(InjectedBrowserWorkbenchEnvironmentService, [], true),
-    [IBrowserWorkbenchEnvironmentService.toString()]: new SyncDescriptor(InjectedBrowserWorkbenchEnvironmentService, [undefined, options], true)
```

Comment:
Is this not needed anymore? I believe I use this to inject product configuration in the extensions gallery PR.

Reply:
IBrowserWorkbenchEnvironmentService is just a refined version of the other one, the toString method return the same